<launch>
  <group ns="robot_1">
    <remap from="left/image_rect" to="/camera/infra1/image_rect_raw"/>
    <remap from="right/image_rect" to="/camera/infra2/image_rect_raw"/>
    <remap from="left/camera_info" to="/camera/infra1/camera_info"/>
    <remap from="right/camera_info" to="/camera/infra2/camera_info"/>
     <node pkg="rtabmap_ros" type="stereo_odometry" name="stereo_odometry" output="screen">
          <param name="frame_id" type="string" value="camera_link"/>
          <param name="estimate_stereo_transform_from_tf" type="bool" value="true"/>
     </node>  
    </group>
</launch>